### PR TITLE
zh: update handle-alerts for newly added panels

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? (Required)

update handle-alerts for https://github.com/pingcap/docs-dm/pull/226

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-dm/pull/226

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: zh/handle-alerts.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
...
+
+存在离线的 DM-worker 超过一小时会触发该告警。在高可用架构下，该告警可能不会直接中断任务，但是会提升任务中断的风险。处理告警可以检查对应 DM-worker 节点的工作状态，是否连通，并检查日志排查错误。
+
+### `DM_DDL_error`
+
+处理 shard DDL 时出现错误，此时需要参考 [DM 故障诊断](error-handling.md#dm-故障诊断)进行处理。
+
+### `DM_pending_DDL`
+
+存在未完成的 shard DDL 并超过一小时会触发该告警，在某些应用场景下，存在未完成的 shard DDL 可能是用户所期望的。在用户预期以外的场景下，可以通过[手动处理 Sharding DDL Lock](manually-handling-sharding-ddl-locks.md)解决。
```
Comment (csuzhangxc) [inline zh/handle-alerts.md:23]:
```suggestion
存在未完成的 shard DDL 并超过一小时会触发该告警。在某些应用场景下，存在未完成的 shard DDL 可能是用户所期望的。在用户预期以外的场景下，可以通过[手动处理 Sharding DDL Lock](manually-handling-sharding-ddl-locks.md)解决。
```

Comment (lance6716) [reply zh/handle-alerts.md]: done

File: zh/handle-alerts.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ aliases: ['/docs-cn/tidb-data-migration/dev/handle-alerts/']
 
 本文档介绍 DM 中各主要告警信息的处理方法。
 
+## 高可用告警
+
+### `DM_worker_offline`
+
+存在离线的 DM-worker 超过一小时会触发该告警。在高可用架构下，该告警可能不会直接中断任务，但是会提升任务中断的风险。处理告警可以检查对应 DM-worker 节点的工作状态，是否连通，并检查日志排查错误。
```
Comment (yikeke) [inline zh/handle-alerts.md:15]:
```suggestion
存在离线的 DM-worker 超过一小时会触发该告警。在高可用架构下，该告警可能不会直接中断任务，但是会提升任务中断的风险。处理告警可以查看对应 DM-worker 节点的工作状态，检查是否连通，并通过日志排查错误。
```

**Pull request merged**